build.gradle.kts

```diff
@@ -56,6 +56,7 @@ dependencies {
   implementation(project("domain:timeline"))
   implementation(project("domain:induction"))
 
+  implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:0.2.2")
```

Comment:
I'm a bit confused with this - why do we need to bring this in? Is it not pulled in via the plugin at the top
```
id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.15.3"
```
?

Are they not one and the same thing? Confused! 🤔

Reply:
My knowledge of Gradle isn't the best so I could be missing something here – but no, they're not the same and I don't believe it's included by the spring boot plugin.

The spring boot plugin is this, https://github.com/ministryofjustice/dps-gradle-spring-boot
The Kotlin spring boot starter library is this, https://github.com/ministryofjustice/hmpps-kotlin-lib

I was mostly just following what other teams have done though, and they have included both,
https://github.com/search?q=org%3Aministryofjustice+implementation%28%22uk.gov.justice.service.hmpps%3Ahmpps-kotlin-spring-boot-starter%3A0.2.1%22%29&type=code

Reply:
Cool, so I follow that they are not the same thing 👍 , but I would have thought the dps-grafle-spring-boot plugin should bring this dependency in for us 🤔 

Perhaps it's related to this comment right at the top of the [readme](https://github.com/ministryofjustice/hmpps-kotlin-lib/blob/main/README.md) of the spring boot starter library?:

> Warning
> This is a work in progress and is not ready for use yet.

Given that comment, and the fact that the library is not included in the main plugin, I wonder if we are are bit premature in doing this? Is it a bit too early for this perhaps?